tests/org.eclipse.jface.tests/src/org/eclipse/jface/tests/viewers/AbstractTreeViewerTest.java

```diff
@@ -164,6 +164,144 @@ public void testExpandToLevel() {
 	}
 
 	@Test
+	public void testAutoExpandTrivialPath() {
```

Comment:
```suggestion
	public void testAutoExpandOnSingleChild() {
```

Reply:
Addressed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/fb256ed9423ce9f552aa278fa3893d833ccc0e63